src/main/java/com/dev/cinema/model/Order.java

```diff
@@ -0,0 +1,37 @@
+package com.dev.cinema.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "order_id")
+    private Long id;
+
+    @ManyToMany
```

Comment:
Why many to many?in 1 order you can have many tickets. But the same ticket can't be on different orders...

Reply:
Yep, agree thx.

Reply:
One particular ticket can be only in one Order. Because order is created from shopping Cart tickets. Each Ticket is related to special user. User can have only one shoppingCart. So in one ShoppingCart we can have a lot of Tickets, but each Ticket can only in one shoppingCart. Same with order.

Reply:
Fixed.